templates/springboot/cloudevents/manifest.yaml

```diff
@@ -1,3 +1,11 @@
+builders:
+  default: gcr.io/paketo-buildpacks/builder:base
+buildEnvs:
+  - name: BP_NATIVE_IMAGE
+    value: "false"
```

Comment:
@salaboy  shouldn't this be false also for spring level manifest that is used for `HTTP` template?

Reply:
also the health endpoint, shouldn't they be set for http template too.

Reply:
> also the health endpoint, shouldn't they be set for http template too.

This implies that the `buildEnv` properties should be set a directory above in `templates/springboot/manifest.yaml`

Reply:
@lance I will send another PR for the HTTP template